src/components/Personalization/constants/scope.js

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
Consider renaming this file to "pageWideScope.js"

Reply:
I have renamed it to `scope` also as part of a review.

Reply:
That was me that [suggested naming it scope.js](https://github.com/adobe/alloy/pull/594/files/66c5115b7517740afa1bb5b78cd23ad3e690536d#r486373572). Just to clarify my stance on this: If the file is intended to potentially contain multiple scopes, then having it called `scope.js` and exporting a constant (`export const PAGE_WIDE_SCOPE = "__view__"`) makes sense, because then other scopes can fall in line as exported constants and it's consistent with https://github.com/adobe/alloy/wiki/Conventions#enumerations.

If the "page wide scope" is the only scope we'll have as a constant (now or in the future), then naming the file `pageWideScope.js` and exporting a default value (`export default "__view__"`) might make more sense sense. 

Exporting the page wide scope as the `default` export value from a file named `scope.js` like we are currently probably doesn't make as much sense in my opinion just because it's less clear that `scope.js` is, specifically, the page wide scope.

Anyway, take it for what it's worth, which is probably not much.